plugins/GCodeReader/FlavorParser.py

```diff
@@ -53,7 +53,7 @@ def __init__(self) -> None:
 
     def _clearValues(self) -> None:
         self._extruder_number = 0
-        self._extrusion_length_offset = [0] # type: List[float]
+        self._extrusion_length_offset = [0,0,0,0] # type: List[float]
```

Comment:
```suggestion
        self._extrusion_length_offset = [0] * 8 # type: List[float]
```
The max nr of extruders supported is 8 (by means of the custom fff printer). The * 8 is a shorthand to write [0, 0, 0, 0, 0, 0, 0, 0]

Reply:
Is there maybe a variable for the number of extruders in the current printer?